### PR TITLE
Enable EKS managed nodegroups and EKS master node to worker node connectivity added

### PR DESCRIPTION
**Pull request by guptavasu1**
Tasks covered in PR
1. Inbound rule created in Node SG allowing all traffic from Master SG.
2. Used Generic Name in Eks Cluster Name
3. Example of EKS managed groups placed in examples/eks_managed_node_group/main.tf
4. Other minor changes are described in the comments.

### CODE REVIEW

**Review by prashantgupta123 — CHANGES_REQUESTED** [review]
1. New SG was created but not used.
2. used your name(ttn-eks-managed-vasu)
3. No description is mentioned in the PR
4. By PR heading it means there is a change in the module but in the changed file, you have added an example, nothing is related to the module issues.

File: main.tf
Comment (guptavasu1) [inline main.tf:1]: resolve_conflicts was deprecated hence updated

File: modules/eks-managed-node-group/main.tf
Comment (guptavasu1) [inline modules/eks-managed-node-group/main.tf:1]: Reached max character limit hence shorten the name.

File: node_groups.tf
Comment (guptavasu1) [inline node_groups.tf:1]: Allowing all traffic from the master to node by adding a rule in node SG

**Pull request merged**